utils/sql/git/required/2021_06_17_server_first_levels.sql

```diff
@@ -0,0 +1,14 @@
+-- ----------------------------
+-- Table structure for server_first_levels
+-- ----------------------------
+CREATE TABLE `server_first_levels` (
```

Comment:
You REALLY need a primary key in this table:

CREATE TABLE IF NOT EXISTS `server_first_levels` (
  `id` int(11) NOT NULL AUTO_INCREMENT,
  `char_name` varchar(64) NOT NULL DEFAULT '',
  `leveled_date` int(11) unsigned NOT NULL DEFAULT 0,
  `race` int(11) NOT NULL DEFAULT 0,
  `class` int(11) NOT NULL DEFAULT 0,
  `level` int(11) NOT NULL DEFAULT 0,
  `account_status` int(11) unsigned NOT NULL DEFAULT 0,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB DEFAULT COLLATE='latin1_swedish_ci';